migrations/829-test-api-key-waffle.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO waffle_switch (name, active, note, created, modified)
+    VALUES ('api-key', 0, 'Waffle for test API key page', NOW(), NOW());
```

Comment:
This switch is too specific. It needs to cover all things related to the signing API. We don't want to add a switch for every API feature because the list of switches will be too long.